tsconfig.json

```diff
@@ -17,6 +17,7 @@
         "noImplicitReturns": true,
         "noFallthroughCasesInSwitch": true,
         "moduleResolution": "node",
+        "verbatimModuleSyntax": true,
```

Comment:
looks ok with the `type` imports https://www.typescriptlang.org/tsconfig/#verbatimModuleSyntax